xapian-core/include/xapian/weight.h

```diff
@@ -29,6 +29,10 @@
 #include <xapian/types.h>
 #include <xapian/visibility.h>
 
+#include "registry.h"
+
+#include "../common/stringutils.h"
```

Comment:
This was needed for `C_isspace` function in the definition of `Weight::create()` in weight.cc. I guess using simply `if (*p = ' ')` might also be fine and then this header will no longer be required.